src/adapters/hasura/group.adapter.ts

```diff
@@ -642,7 +642,7 @@ export class HasuraGroupService implements IServicelocatorgroup {
       variables: {
         schoolUdise: schoolUdise,
         name: className,
-        year: year
+        year: year,
```

Comment:
@Xitija is this extra comma?

Reply:
Mam it doesnt cause issue it is not extra